interface/patient_file/transaction/transactions.php

```diff
@@ -56,8 +56,8 @@ function deleteme(transactionId) {
 ?>
 </head>
 
-<body class="body_top">
-    <div id="container_div" class="<?php echo $oemr_ui->oeContainer();?>">
+<body>
+    <div id="container_div" class="<?php echo $oemr_ui->oeContainer();?> mt-3">
         <div class="row">
```

Comment:
Add some spacing between the `btn`'s and the navbar to prevent them from looking like they're colliding
![image](https://user-images.githubusercontent.com/46497068/91072306-4bdaa000-e607-11ea-8acb-609b6b5a6d36.png)